src/components/organisms/Room/Room.tsx

```diff
@@ -81,13 +76,10 @@ export const Room: React.FC<RoomProps> = ({
 
         return (
           <div
-            key={participant.participant.identity}
+            key={participant.participant.sparkleId}
             className={`participant-container ${participantContainerClassName}`}
           >
-            <VideoParticipant
-              participant={participant.participant}
-              participantUser={participant.user}
-            />
+            <VideoCommsParticipant participant={participant.participant} />
```

Comment:
`Spam spam={spam.spam}` 🤷 hehe

`Participant participant={participant.participant}`

Reply:
yeah :( Trying to do the minimal refactoring to get this new system embedded without making the merge back into the design branch hell.